src/tox/_pytestplugin.py

```diff
@@ -69,7 +69,7 @@ def cmd(request, capfd, monkeypatch):
     request.addfinalizer(py.path.local().chdir)
 
     def run(*argv):
-        key = str(b"PYTHONPATH")
+        key = b"PYTHONPATH" if six.PY2 else "PYTHONPATH"
         python_paths = (i for i in (str(os.getcwd()), os.getenv(key)) if i)
```

Comment:
hmm, I can't think of a situation where `getcwd()` returns non-native string -- I suspect the `str(...)` call here is superfluous